utils/forms.py

```diff
@@ -38,23 +42,27 @@ def clean(self, value):
         value = super(HtmlCleaningCharField, self).clean(value)
         if is_shouting(value):
             raise forms.ValidationError('Please moderate the amount of upper case characters in your post...')
-        try:
-            return clean_html(value)
-        except (HTMLParseError, UnicodeEncodeError):
```

Comment:
after discussion with @andrebola we realised that the switch to bleach no longer raises these exceptions